cms/templates/js/mock/mock-settings-page.underscore

```diff
@@ -105,6 +105,129 @@
       </select>
       <span class="tip tip-stacked">Identify the course language here. This is used to assist users find courses that are taught in a specific language.</span>
     </li>
+
   </ol>
 </section>
+
+<section class="group-settings marketing">
+  <header>
+    <h2 class="title-2">${_("Introducing Your Course")}</h2>
```

Comment:
Note that in general, testing (mocked) files should not have i18n calls. The issue is that we could end up wasting translating manpower on test strings. In this case (where it is a copy of the source file and thus introduces no new strings), I don't feel strongly about it. However, it is likely that these 2 files will end up drifting, which is the bigger issue with a mocked page.